nix/default.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs, lib, buildGoModule, git_version, ... }: let
+  manifest = builtins.fromJSON (builtins.readFile (../. + "/tlock.json"));
```

Comment:
You are referring to the scoop manifest to get the pkg version. Rather than this, how about getting the latest tag (since it will be the newest version) with `git describe --tags --abbrev=0`?

I don't have much experience with Nix myself, if you think that this would not be a great idea, let me know :)

Reply:
From what I can see hercules CI runs when deploying to a machine.
I haven't been able to access the option in the package definition.

I just made an assumption that the manifest would reflect the release version :x I used it so you hopefully wouldn't have to change it in multiple places.
I can totes just hardcode it if you'd prefer that~

It also got me thinking.. Would you prefer that it builds from the release rather than the current revision?
I prefer to use a relative path to the local revision, so that it builds itself and you don't have to specify and update a revision. It also makes building local changes with nix easy, but it depends on your workflow and whether you expect the main branch to always be in a working state.

Reply:
>  I can totes just hardcode it if you'd prefer that~

Yeah, I think that would be good if you cant access the package definition. That is true that the manifest will reflect the latest version, but I just wanted to prevent coupling stuff with these manifests because I might just move them to their main repos one day and forget that some other files were still referring to it if that makes sense (Im bad at explaining stuff)

> Would you prefer that it builds from the release rather than the current revision?

Its fine..... let us keep this way then. I do expect that the master branch will always be stable :)